test/test_printing.jl

```diff
@@ -24,7 +24,8 @@ function are_type_and_fields_in_output(obj::T) where {T <: Component}
 
     for (fieldname, fieldtype) in zip(fields, fieldtypes(T))
         if isnothing(getfield(obj, fieldname)) ||
-           fieldtype <: IS.InfrastructureSystemsInternal
+           fieldtype <: IS.InfrastructureSystemsInternal ||
+           fieldtype <: PowerSystems.OperationalCost
```

Comment:
@daniel-thom I added this because I couldn't figure out how to extend the conditional in line 39 to catch all subtypes. Let me know if you have other ideas.

Reply:
There are likely two problems:
1. Sometimes the field type is `OperationCost` which won't show up.
2. Sometimes the field type is `Union{Nothing, OperationCost}`

Try changing line 34 to
```
val = getfield(obj, fieldname)
expected = string(typeof(val))
```
Assignment of `val` could be made at the top of the for loop.